front/pages/w/[wId]/u/chat/[cId]/index.tsx

```diff
@@ -841,6 +859,34 @@ export default function AppChat({
     while (m.pop()?.role !== "user" && m.length > 0);
   }
 
+  function logFeedback(message: ChatMessageType): void {
+    const chatSessionUrl = `https://dust.tt/w/${owner.sId}/u/chat/${chatSession.sId}`;
+
+    // gets the user message preceding `message` in the chat session
+    const messagesCopy = [...messages];
+    let previousUserMessage: ChatMessageType | undefined;
+    while (messagesCopy.pop() !== message);
+    while ((previousUserMessage = messagesCopy.pop())?.role !== "user");
+
+    // create the logging message
+    const loggingMessage = {
+      topic: "Feedback",
```

Comment:
A datadog message is a "message" and some {} attributes so the signature need to change here as well cf other comment.

More precisely on the string message. "feedback" is not precise enough. We're in the chat app and the feedback is on a message Something like "Chat message Feedback" would be better suited as we'll get feedback from other products in the near future